package.json

```diff
@@ -156,6 +156,19 @@
   },
   "license": "GPL-3.0-or-later",
   "dependencies": {
+    "@ethersproject/abi": "^5.4.1",
```

Comment:
slightly concerned w the diff patch versions. maybe just set them all to the same one and do a yarn install?

Reply:
I didn't want to change any versions in this PR, so I copied the pinned versions that the `ethers` package was using, prepended them with `^` for compatibility, and manually edited the `yarn.lock` to make sure the versions weren't bumped. Changing patch versions to all match would effectively change the versions we use.